collect_app/src/main/java/org/odk/collect/android/widgets/BarcodeWidget.java

```diff
@@ -57,7 +58,14 @@ public void onClick(View v) {
 
                 Collect.getInstance().getFormController()
                         .setIndexWaitingForData(formEntryPrompt.getIndex());
-                new IntentIntegrator((Activity) getContext()).initiateScan();
+
+                new IntentIntegrator((Activity) getContext())
+                        .setCaptureActivity(ScannerWithFlashlightActivity.class)
+                        .setBeepEnabled(true)
```

Comment:
I think that always having a beep is fine for the settings QR code because it will be used by project organizers once. In this context where it will generally be used in a data collection context around other people, I think it could be disruptive. What do you think @kusaasira, @shobhitagarwal1612?

Reply:
@lognaturel You are right. I have removed that for now.

Reply:
@lognaturel I somehow do not side with the disruptive idea. I think it confirms that the scan is successful. How is that? If we look at it from an informative end.

Reply:
I see what you mean, @kusaasira, but I think that since the user goes back to the form on successful scan it's pretty clear. ODK Collect is used in locations like hospitals or libraries where beeping really could be a problem!